modules/directives/src/main/scala/scala/build/preprocessing/directives/ResourceFiles.scala

```diff
@@ -0,0 +1,65 @@
+package scala.build.preprocessing.directives
+
+import scala.build.Positioned
+import scala.build.directives.*
+import scala.build.errors.BuildException
+import scala.build.options.WithBuildRequirements.*
+import scala.build.options.{BuildOptions, ClassPathOptions, Scope, WithBuildRequirements}
+import scala.cli.commands.SpecificationLevel
+
+@DirectiveGroupName("Resource files")
+@DirectiveExamples("//> using resourceFile ./resource.txt")
+@DirectiveExamples("//> using test.resourceFile ./resource.txt")
+@DirectiveUsage(
+  """//> using resourceFile _path_
+    |
+    |//> using resourceFiles _path1_ _path2_ …""".stripMargin,
+  """`//> using resourceFile` _path_
+    |
+    |`//> using resourceFiles` _path1_ _path2_ …""".stripMargin
+)
+@DirectiveDescription("Manually add a resource files to the class path")
+@DirectiveLevel(SpecificationLevel.SHOULD)
+final case class ResourceFiles(
```

Comment:
Rather than create a separate directive declaration, maybe can reuse `scala.build.preprocessing.directives.Resources`, directives for both directories and files can be there.
Refer to `ScalaNative` directives, which are all declared in one source file.

Reply:
Alternatively (optionally) maybe the code for the existing directive could be modified to detect if the passed path is for a directory or a single file, and process it accordingly (a separate directive is also ok though). Then it'd be ok to alias the name of the old directive (and keep the `resourceDir` name for backwards compatibility)